HCaptcha/Assets/journeys.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://hcaptcha.com/schemas/ios/journeys.schema.json",
+  "title": "HCaptcha Journeys Events",
+  "type": "array",
+  "items": {
```

Comment:
Is there any value to using single characters or acronyms instead of defining the properties fully?